### PR TITLE
Stop current frame when starting something else

### PR DESCRIPTION
**Pull request by waynew**
I don't know if it's just the way I work, but I dislike having to say

    $ watson stop
    $ watson start <new project> [new tags]

So I now I don't have to :)

### CODE REVIEW

**Comment by jmaupetit** [comment]
I am not sure this must be the default behavior. An option or global configuration must be set for that.

**Comment by k4nar** [comment]
@jmaupetit : :+1:

**Comment by waynew** [comment]
@jmaupetit that certainly makes sense. I don't see any examples in cli.py that actually *use* any kind of config (except for the config, which has to load up the config first), so I don't think I have any guidance to go on for how to approach that...

**Comment by SpotlightKid** [comment]
@waynew  Look at PR 36 and PR 43

**Comment by waynew** [comment]
That's #36 and #43 for the lazy :)

**Comment by waynew** [comment]
Thanks @SpotlightKid.

---

Updated to use the config, but this *does* depend on #43 being merged in or it will break, absent that config in the file.

I used the setting `auto_stop` because it was the first and most accurate setting name that came to mind that wasn't ridiculously long like `stop_current_task_when_starting_another`. I don't have strong feelings about the former if there's something else concise and maybe more descriptive.

**Comment by jmaupetit** [comment]
What about `stop_on_start`?

**Comment by waynew** [comment]
I need to change this to use `.getboolean`, as #43 will parse `stop_on_start = False` correctly. I think how it is that would probably just evaluate to `'False'`, which is `True` :trollface:

**Comment by k4nar** [comment]
This should be rebased now that #43 is merged.

**Comment by waynew** [comment]
@k4nar done

Also did the `.get` -> `.getboolean` thing

**Comment by jmaupetit** [comment]
:+1:

**Comment by k4nar** [comment]
@waynew : Great, thanks !

Could you update the docstring and the readme to document this new option ?

File: watson/cli.py [hunk 1]
```diff
@@ -119,6 +119,18 @@ def start(watson, args):
     $ watson start apollo11 +module +brakes
     Starting apollo11 [module, brakes] at 16:34
     """
+    if watson.config.getboolean('options', 'stop_on_start'):
```
Comment (k4nar) [inline watson/cli.py]: We should check `watson.is_started` instead of using a try/except.

**Comment by SpotlightKid** [comment]
@waynew FWIW, since I included the feature in #49 as well, I added documentation to the README and `cli.start` there already.

**Comment by waynew** [comment]
@k4nar given that @SpotlightKid already added some docs, should I just copy those? Not sure if there's a reasonable way to get that particular commit in my branch :P

**Comment by k4nar** [comment]
@waynew : Yes you can copy the docs. When @SpotlightKid will rebase, it will no longer be in his commit :) .

**Comment by jmaupetit** [comment]
@k4nar Ready to be merged?

**Comment by k4nar** [comment]
@jmaupetit : I would like to see some docs with that PR.

**Comment by SpotlightKid** [comment]
@waynew Did you have time to include the docs yet? I want to present Watson at our local user group meeting next week (Wed) and it would be cool if this PR and #49 could be merged by then!

**Comment by waynew** [comment]
@SpotlightKid / @k4nar / @jmaupetit Docs :+1:

**Comment by SpotlightKid** [comment]
@waynew Thanks!

**Comment by k4nar** [comment]
Great, thank you for your time :+1:!

**Comment by jmaupetit** [comment]
@waynew you rock! :+1:

**Comment by jmaupetit** [comment]
@SpotlightKid Please let us know if you have any feedback from your talk!

**Pull request merged**